staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -39,10 +39,11 @@ const duration365d = time.Hour * 24 * 365
 
 // Config contains the basic fields required for creating a certificate
 type Config struct {
-	CommonName   string
-	Organization []string
-	AltNames     AltNames
-	Usages       []x509.ExtKeyUsage
+	CommonName         string
+	Organization       []string
+	AltNames           AltNames
+	Usages             []x509.ExtKeyUsage
+	PublicKeyAlgorithm x509.PublicKeyAlgorithm
```

Comment:
Do we really need this setting here? We can create a kubeadm wrapper struct that embeds this one and adds the `PublicKeyAlgorithm` field.